pdc/apps/common/viewsets.py

```diff
@@ -205,7 +205,7 @@ def _check_ordering_keys(self, request):
         serializer_fields = self._get_fields_from_serializer_class()
         valid_fields = list(set(model_fields).intersection(set(serializer_fields)))
         valid_fields += self.queryset.query.aggregates.keys()
-        tmp_list = [param.strip().lstrip('-') for param in ordering_keys.split(',')]
+        tmp_list = [param.strip().lstrip('-').split("__")[0] for param in ordering_keys.split(',')]
```

Comment:
Hmmm, in my local env, it can raise error, a little strange：
 pdc_client -s http://<server>:8080/rest_api/v1 -r release-component-relationships -d '{"ordering":"from_component__zzz"}'

500 b'\n<!DOCTYPE html>\n<html lang="en">\n<head>\n  <meta http-equiv="content-type" content="text/html; charset=utf-8">\n  <meta name="robots" content="NONE,NOARCHIVE">\n  <title>FieldError at /rest_api/v1/release-component-relationships/</title>\n  <style type="text/css">\n    html * { padding:0; margin:0; }\n    body * { padding:10px 20px; }\n    body * * { padding:0; }\n    body { font:small sans-serif; }\n    body>div { border-bottom:1px solid #ddd; }\n    h1 { font-weight:normal; }\n    h2 { margin-bottom:.8em; }\n   ........

 pdc_client -s http://<server>:8080/rest_api/v1 -r release-component-relationships -d '{"ordering":"from_component__name"}'
{
    "count": 3,
    "next": null,
    "previous": null,
    "results": [
        {
            "from_component": {
                "id": XXX,
                "name": "XXX",
                "release": "XXXX"
            },
......